PayGreen/paygreen-ui
# PGF-68 - Amélioration ClickableBlock

## How to test ?

- Launch the storybook
- Go to section ClickableBlock
- Check that : 
  - the ClickableBlock colors props works (Pallet, Theme, Wab and Status)
  - the props margin and padding works
  - the props iconSize works
  - the onClick no longer works
  - the background opacity is sufficient on hover, focus and active
